Source/CameraView/CameraView.swift

```diff
@@ -218,7 +218,8 @@ class CameraView: UIViewController, CLLocationManagerDelegate, CameraManDelegate
 
   func takePicture(_ completion: @escaping () -> Void) {
     guard let previewLayer = previewLayer else { return }
-
+    SoundEffect.cameraShutter.play()
```

Comment:
Should this be wrapped in a check against the configuration? 

```Swift
if configuration.addCameraSoundEffect {
  SoundEffect.cameraShutter.play()
}

```

Reply:
Oh yeah, good catch. I should really review more code more often 👍